bpf-prog/bpf-tc/linux.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/oss/go/microsoft/golang:1.21 AS builder
+ARG VERSION
+WORKDIR /bpf-prog/bpf-tc
+COPY ./bpf-prog/bpf-tc .
+COPY ./bpf-prog/bpf-tc/cmd/bpf-tc/*.go /bpf-prog/bpf-tc/
+RUN apt-get clean
+RUN apt-get update && apt-get install -y llvm clang linux-libc-dev linux-headers-generic libbpf-dev libc6-dev gcc-multilib nftables iproute2
+RUN for dir in /usr/include/x86_64-linux-gnu/*; do ln -s "$dir" /usr/include/$(basename "$dir"); done
```

Comment:
why this is required?

Reply:
several of the header files are not available when compiling in the container, we need this to to create symlinks from /usr/include/x86_64-linux-gnu to directories in /usr/include so that everything can be compiled